sys/include/net/ble_ll.h

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        ble_ll BLE Link Layer
+ * @ingroup         ble_ll
+ * @brief           Ble link layer implementation
+ * @{
+ *
+ * @file
+ * @brief           Interface definition for the ble link layer
+ *
+ * @author          Jan Wagner <mail@jwagner.eu>
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef BLE_LL_H_
+#define BLE_LL_H_
+
+#include "net/ng_netdev.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Default channel set after initialization
+ */
+#define BLE_DEFAULT_CHANNEL         37  /* 2402MHz */
+
+/**
+ * @brief   Default transmission power used
+ */
+#define BLE_DEFAULT_TXPOWER         0 /* 0dBm */
+
+/**
+ * @brief   BLE Advertising channel PDU types
+ * @{
+ */
+enum pdu_types {
+    ADV_IND_TYPE,
+    ADV_DIRECT_IND_TYPE,
+    ADV_NONCONN_IND_TYPE,
+    SCAN_REQ_TYPE,
+    SCAN_RSP_TYPE,
+    CONNECT_REQ_TYPE,
+    ADV_SCAN_IND_TYPE
+};
+/** @} */
+
+/**
+ * @brief   BLE constant lengths and limits
+ * @{
+ */
+#define BLE_MAX_CHANNELS            39
+#define BLE_PDU_HDR_LEN             2
+#define BLE_ACCESS_ADDR_LEN         4
+#define BLE_ADDR_LEN                6
+#define BLE_PAYLOAD_LEN_MIN         (BLE_ADDR_LEN)
+#define BLE_PAYLOAD_LEN_MAX         37
+#define BLE_MAX_DATA_LEN            (BLE_PAYLOAD_LEN_MAX - BLE_ADDR_LEN)
+#define BLE_MAX_PKT_LEN             275
+
```

Comment:
Like privately discussed, could you also bring some light to these constants? maybe with a small comment or so